pom.xml

```diff
@@ -116,7 +116,7 @@
 
         <!-- logging properties - we expect logging jars to be provided -->
         <!-- 1.7.x is for hadoop 2 and accumulo 1.5.x -->
-        <slf4j.version>1.7.5</slf4j.version>
+        <slf4j.version>1.7.21</slf4j.version>
```

Comment:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=15184 (1.7.21)

Reply:
we'll need to open piggybacks, right?

Reply:
i think this is OK. it is likely baselined on accumulo 1.8.1

Reply:
Approved: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16001